dev/com.ibm.ws.kernel.metatype.helper/src/com/ibm/wsspi/config/internal/FilesetImpl.java

```diff
@@ -541,4 +573,29 @@ private void notifyListeners() {
         }
     }
 
+    @Override
+    @FFDCIgnore(Exception.class)
+    public String toString() {
+        try {
+
+            StringBuilder sb = new StringBuilder();
+
+            sb.append("FileSet for directory: " + basedir);
+            sb.append(System.lineSeparator());
+
+            if (filter.hasFilters()) {
+                sb.append(filter.getHumanReadableSummary()); //provides its own line break
+            }
+
+            String files = fileset.stream().map(File::getPath).collect(Collectors.joining(", "));
+            sb.append("Which found the following files: " + files);
```

Comment:
quibble: again, use the buffer for string concatenation
`sb.append("Which found the following files: ").append(files)`